exercises/practice/isogram/.meta/example.zig

```diff
@@ -1,23 +1,12 @@
-pub fn isIsogram(str: []const u8) bool {
-    var mask: u32 = 0;
-    for (str) |value| {
-        switch (value) {
-            'a'...'z' => {
-                if (mask & @as(u32, 1) << @as(u5, @truncate(value - 'a')) == 0) {
-                    mask |= @as(u32, 1) << @as(u5, @truncate(value - 'a'));
-                } else {
-                    return false;
-                }
-            },
-            'A'...'Z' => {
-                if (mask & @as(u32, 1) << @as(u5, @truncate(value - 'A')) == 0) {
-                    mask |= @as(u32, 1) << @as(u5, @truncate(value - 'A'));
-                } else {
-                    return false;
-                }
-            },
+pub fn isIsogram(s: []const u8) bool {
+    var letters = [_]bool{false} ** 26;
+    for (s) |c| {
+        const i = switch (c) {
+            'A'...'Z' => c - 'A',
+            'a'...'z' => c - 'a',
             else => continue,
-        }
+        };
+        if (letters[i]) return false else letters[i] = true;
```

Comment:
```suggestion
        if (letters[i]) return false;
        letters[i] = true;
```

Reply:
Very minor, take it or leave it, but for readability I might choose to break this up.

One is an exit criteria for the function, and another is a flag for future iterations, so it feels like they're doing fairly different things.